app/controllers/events_controller.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+class EventsController < ApplicationController
+  def index
+    @events = Event.includes(:creator).all
+  end
+
+  def new
+    @event = Event.new
+  end
+
+  def create
+    if current_user.events.create(event_params)
```

Comment:
According to the `create` method [documentation](https://apidock.com/rails/ActiveRecord/Base/create/class), `create` will return an object whether the object was saved successfully to the database **or not** so it is better to use the `build` method before and in the condition, you can use the `save` method to check if the object was saved (returning `true` or `false`).